src/Contact.scss

```diff
@@ -2,9 +2,30 @@
     text-align: left;
     padding-inline-start: 0px;
     background-color: #FFF;
+    flex-shrink: 0;
     li {
         list-style-type: none;
         margin-top: 10px;
     }
+
+    @media (max-width: 900px) {
+        width: 377px;
+        height: 202px;
+        display: flex;
+        flex-direction: column; 
+        align-items: flex-start;
+        
+    }
+
+    @media (min-width: 901px) {
+        display: flex;
+        flex-direction: row; 
+        justify-content: space-around;
+        align-items: center;
+        display: flex;
+        padding: 31px 397px;
```

Comment:
we are not using padding this big because it will not fit to all of the px screens

Reply:
fixed